.devcontainer/devcontainer.json

```diff
@@ -1,25 +1,44 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/azure-bicep
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/dotnet
 {
-	"name": "Azure Bicep (Community)",		
-	"dockerFile": "Dockerfile",
-	
+	"name": "Mock Space Station Dev Container",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update 'VARIANT' to pick a .NET Core version: 2.1, 3.1, 5.0
+			"VARIANT": "5.0",
+			// Options
+			"NODE_VERSION": "none",
+			"INSTALL_AZURE_CLI": "false"
+		}
+	},
+	"runArgs": [
+		"--init",
+		"--privileged"
+	],
+	"overrideCommand": false,
 	// Set *default* container specific settings.json values on container create.
 	"settings": {},
-		
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
-		"bierner.markdown-preview-github-styles",
-		"davidanson.vscode-markdownlint",
-		"ms-vscode.azurecli",
-		"ms-dotnettools.vscode-dotnet-runtime",
-		"ms-azuretools.vscode-bicep",
-		"timonwong.shellcheck"
+		"ms-dotnettools.csharp"
```

Comment:
Any reason to change the extensions installed? I think the ones removed here are still helpful and relevant for authoring this repo. I also don't see any reason to use the C# extension?